addons_rksv/pos_rksv/views/pos_config.xml

```diff
@@ -16,7 +16,8 @@
                     </header>
                 </xpath>
                 <field name="iface_cashdrawer" position="after">
-                    <field name="iface_rksv" />
+                    <field name="rksv_at" invisible="True"/>
+                    <field name="iface_rksv" attrs="{'invisible': [('rksv_at', '=', False)]}"/>
```

Comment:
Do we have a mechanism which will set this to ```False``` in case rksv_at was disabled later?